bokehjs/test/test_common/test_selection_manager.coffee

```diff
@@ -77,9 +77,6 @@ describe "SelectionManager", ->
 
   sm = new SelectionManager()
 
-  it "should have a reference to an empty selection", ->
-    expect(sm.empty).to.deep.equal empty_selection
```

Comment:
@birdsarah, in the context of our yesterday's discussion, here's a nice example of a test that does nothing, yet exposes internal implementation of the class and gets in the way of modifying it. Additionally, `empty` should be a private member of `SelectionManager`.